src/community/infix-short-scale.ts

```diff
@@ -0,0 +1,29 @@
+import { AbstractInfixNotation } from "./infix-abstract";
+import { toSubscript, abbreviate } from "../utils";
+import type Decimal from "break_infinity.js";
+
+// Name comes from https://en.wikipedia.org/wiki/Long_and_short_scales
+export class InfixShortScaleNotation extends AbstractInfixNotation {
+  public readonly name: string = "Infix short scale";
+
+  public formatNegativeDecimal(value: Decimal, places: number): string {
+    return `₋${this.formatDecimal(
+      value,
+      places
+    )}`;
+  }
+
+  protected canHandleZeroExponent = false;
```

Comment:
Codacy found an issue: [Declaration of protected instance field not allowed after declaration of public instance method. Instead, this should come after public instance fields.](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)